docs/components/GestaltLogo.js

```diff
@@ -0,0 +1,33 @@
+// @flow strict
+import { type Node } from 'react';
+
+type Props = {|
+  height?: number,
```

Comment:
@dangerismycat Nit & out of scope for this PR

What do you think about making Gestalt props read only? It would
* encourage best practices
* avoid certain bugs

2 ways we could do this:

* `$ReadOnly` utility type: https://flow.org/en/docs/types/utilities/#toc-readonly
* Add a `+` for every prop: https://flow.org/en/docs/types/interfaces/#toc-interface-property-variance-read-only-and-write-only

Reply:
Sounds like a great idea, but I want to talk through any trade-offs / side effects with the rest of the team. I added it to our agenda for next week's Eng meeting.